opus/application/apps/cart/views.py

```diff
@@ -72,8 +74,11 @@ def api_view_cart(request):
 
     This is a PRIVATE API.
 
-    Format: __cart/view.html
+    Format: __cart/view.json
```

Comment:
Ooops one more change - should change the line in the main file header that lists the API calls in this file.